CondFormats/SiPixelTransient/interface/SiPixelTemplate2D.h

```diff
@@ -101,13 +101,8 @@ struct SiPixelTemplateHeader2D {  //!< template header structure
 
 struct SiPixelTemplateStore2D {  //!< template storage structure
   SiPixelTemplateHeader2D head;
-  SiPixelTemplateEntry2D** entry = nullptr;  //!< use 2d entry to store BPix and FPix entries [dynamically allocated]
-  void destroy() {                           // deletes arrays created by pushfile method of SiPixelTemplate
-    if (entry != nullptr) {
-      delete[] entry[0];
-      delete[] entry;
-    }
-  }
+  std::vector<std::vector<SiPixelTemplateEntry2D>>
+      entry;  //!< use 2d entry to store BPix and FPix entries [dynamically allocated]
```

Comment:
please move the comment above the code to avoid the line split

Reply:
Ok